src/utils/apis/LoginAPI.js

```diff
@@ -21,7 +21,6 @@ export const loginAPI = {
             })
         }).then(response => {
             jwtUtils.saveTokenFromResponse(response)
-            setInterval(jwtUtils.refreshToken, 2000)
```

Comment:
Nie potrzebujemy tego interval?

Reply:
Nie, napisałem na dc na ogólnym jak to działa tera